site/docs/master/output-file-format.md

```diff
@@ -67,33 +67,96 @@ resources/
         cluster/
             pv01.json
             ...
+        v1-preferredversion/
+            cluster/
+                pv01.json
+                ...
     configmaps/
         namespaces/
             namespace1/
                 myconfigmap.json
                 ...
             namespace2/
                 ...
+        v1-preferredversion/
+            namespaces/
+                namespace1/
+                    myconfigmap.json
+                    ...
+                namespace2/
+                    ...
     pods/
         namespaces/
             namespace1/
                 mypod.json
                 ...
             namespace2/
                 ...
-    jobs/
+        v1-preferredversion/
+            namespaces/
+                namespace1/
+                    mypod.json
+                    ...
+                namespace2/
+                    ...
+    jobs.batch/
         namespaces/
             namespace1/
                 awesome-job.json
                 ...
             namespace2/
                 ...
+        v1-preferredversion/
+            namespaces/
+                namespace1/
+                    awesome-job.json
+                    ...
+                namespace2/
+                    ...
     deployments/
         namespaces/
             namespace1/
                 cool-deployment.json
                 ...
             namespace2/
                 ...
+	v1-preferredversion/
+		namespaces/
+		    namespace1/
+			cool-deployment.json
+			...
+		    namespace2/
+			...
+    horizontalpodautoscalers.autoscaling/
+        namespaces/
+            namespace1/
+                hpa-to-the-rescue.json
+                ...
+            namespace2/
+                ...
+        v1-preferredversion/
+            namespaces/
+                namespace1/
+                    hpa-to-the-rescue.json
+                    ...
+                namespace2/
+                    ...
+        v2beta1/
+            namespaces/
+                namespace1/
+                    hpa-to-the-rescue.json
+                    ...
+                namespace2/
+                    ...
+        v2beta2/
+            namespaces/
+                namespace1/
+                    hpa-to-the-rescue.json
+                    ...
+                namespace2/
+                    ...
+
     ...
 ```
+
+Recently, we have added all API groups versions part of the backup (previously, only preferred version of each API Groups was backed up). Each resource has one or more sub-directories, one sub-directory for each supported version of the API group. The preferred version API Group of each resource has the suffix "-preferredversion" as part of the sub-directory name. For backward compatibility, we kept the classic directory structure without API Group version, which sits on the same level of the API Group sub-directory versions.
```

Comment:
@skriss @carlisia @ashish-amarnath I'm curious if you all think we should bump our format to v1.1 with this change or not. It's backwards compatible, but this would signal that there's a change, and be more explicit that there's new features. It would also mean that this paragraph could change from `Recently`, which is context-sensitive, to `In backup format v1.1, we have added...`, which is not.

Reply:
That's a good point. It makes sense to me. But I'm also not clear on what should trigger a bump, maybe we could take a moment to discuss.

Reply:
@skriss @ashish-amarnath What are your thoughts on this?

Reply:
I'd like to get this resolved, so I've added this to the April 14 2020 community meeting discussion topics. IMO, this is the only thing block merge at this point.

Reply:
Discussed and resolved - we'll do a minor version bump.